packages/ckeditor5-dev-env/lib/release-tools/utils/versions.js

```diff
@@ -17,11 +17,19 @@ const versions = {
 	 */
 	getLastFromChangelog() {
 		const changelog = changelogUtils.getChangelog();
-		const regexp = /\n## \[?([^\]]+)/;
 
+		if ( !changelog ) {
+			return null;
+		}
+
+		const regexp = /\n## (\[([^\]]+)|[^ ]+)/;
 		const matches = changelog.match( regexp );
 
-		return matches ? matches[ 1 ] : null;
+		if ( !matches ) {
+			return null;
+		}
+
+		return matches[ 2 ] ? matches[ 2 ] : matches[ 1 ];
```

Comment:
Now we handle both types of changelog entry:

1. For entry generated for the first time:
```md
CHANGELOG
==========

## version (date)
```

2. For the next entries:
```md
CHANGELOG
==========

## [version](URL to compare the changes) (date)
```